draft-ietf-teas-actn-poi-applicability.txt

```diff
@@ -1332,25 +1324,46 @@ Internet-Draft                 ACTN POI                    January 2023
 
 4.1. Optical Topology Discovery
 
-   The WSON Topology Model or, alternatively, the Flexi-grid Topology
-   model is used to report the DWDM network topology (e.g., ROADM nodes
-   and links), depending on whether the DWDM optical network is based on
-   fixed grid or flexible-grid.
+   The WSON Topology Model and the Flexi-grid Topology model can be used
+   to report the DWDM network topology (e.g., ROADM nodes and links),
+   depending on whether the DWDM optical network is based on fixed-grid
+   or flexible-grid or a mix of fixed-grid and flexible-grid.
 
-   The OTN Topology Model is used to report the OTN network topology
-   (e.g., OTN switching nodes and links), when the OTN switching layer
-   is deployed within the optical domain.
+   It is worth noting that, as described in xxx, a fixed-grid can also
+   be described as a flexible grid with constraints: for example a 50GHz
+   fixed-grid can be described as a flexible-grid which supports only
+   m=4 and values of n which are only multiplier of 8.
 
-   To allow the MDSC to discover the complete multi-layer and multi-
-   domain network topology and to correlate it with the hardware
+   As a consequence:
+
+   o  A flexible-grid DWDM network topology can only be reported using
+      the Flexi-grid Topology model;
 
+   o  A fixed-grid DWDM network topology, can be reported using either
+      the WSON Topology model or the Flexi-grid Topology model;
 
 
-Peruzzini et al.        Expires July 11, 2023                 [Page 25]
+
+
+
+Peruzzini et al.      Expires December 19, 2023               [Page 25]
 
-Internet-Draft                 ACTN POI                    January 2023
+Internet-Draft                 ACTN POI                       June 2023
 
 
+   o  A mixed fixed and flexible grid DWDM network topology can be
+      reported using either the Flexi-grid Topology model or both WSON
+      and Flexi-grid topology models.
+
+   Clarifying how both WSON and Flexi-grid topology models could be used
+   together (through multi-inheritance) has been identified as a gap.
+
```

Comment:
Just a suggestion to add a reference to the concept of multi-inheritance to whom is not familiar with these models. (maybe RFC 8345) or draft-busi-teas-te-topology-profile

Reply:
**2023-07-04 TEAS Call**

Agreed to add an informative reference to draft-busi-teas-te-topology-profile

Reply:
Fixed in https://github.com/IETF-TEAS-WG/actn-poi/pull/114/commits/65ec90d608f752d341a829c66e428d418c45da90